ecs-cli/modules/utils/compose/convert_task_definition.go

```diff
@@ -101,6 +102,13 @@ func ConvertToTaskDefinition(taskDefinitionName string, context *project.Context
 		volumeWithHost: make(map[string]string), // map with key:=hostSourcePath value:=VolumeName
 	}
 
+	// Add named volume configs:
+	if context.Project.VolumeConfigs != nil {
+		for name := range context.Project.VolumeConfigs {
```

Comment:
if any of the Volumes include non-`name` args, (like `driver`, `driver_opts`, etc.) let's log a warning re: Skipping an unsupported option